### PR TITLE
check for test coverage

### PR DESCRIPTION
**Pull request by tanzr**

### CODE REVIEW

**Comment by oyvindkolbu** [comment]
Missing dependencies in requirements?

File: .travis.yml [hunk 1]
```diff
@@ -23,4 +23,4 @@ before_script:
   - psql -c 'create database travisci;' -U postgres
   - python manage.py migrate
 script:
-  - python manage.py test
\ No newline at end of file
+  - python manage.py test
```
Comment (oyvindkolbu) [inline .travis.yml]: Should this be changed to `coverage run mange.py test`?

Comment (tanzr) [reply .travis.yml]: If you see in the first PR, I added (coverage run manage.py test & coverage report), but travis failed on its checks.

Comment (oyvindkolbu) [reply .travis.yml]: Probably because of missing dependencies in `requirements.txt`.

File: .travis.yml [hunk 2]
```diff
@@ -23,4 +23,4 @@ before_script:
   - psql -c 'create database travisci;' -U postgres
   - python manage.py migrate
 script:
-  - python manage.py test
+  - coverage run mange.py test
```
Comment (oyvindkolbu) [inline .travis.yml]: https://travis-ci.org/usit-gd/mreg/builds/511881091#L478

**Comment by oyvindkolbu** [comment]
Resolves #269.

**Pull request merged**